src/ServiceBouncer/MainForm.cs

```diff
@@ -15,22 +17,37 @@ public partial class MainForm : Form
         private readonly List<ServiceViewModel> services;
         private bool isActive;
         private string machineHostname;
-        private int backgroundRefreshSeconds = 1;
+        private int backgroundRefreshSeconds;
 
-        public MainForm()
+        public MainForm(Options options = null)
```

Comment:
Could you pass machine name in here.
If there is no options, then use the default of current machine.